src/components/projects-summary/projects-summary.html

```diff
@@ -1,11 +1,75 @@
+  </div>
+
+  <div ng-if="$ctrl.showGetStarted">
+    <h2>Getting Started</h2>
+    <div class="getting-started-panel">
+      {{$ctrl.gettingStartedText}}
+    </div>
+  </div>
+  <div ng-if="!$ctrl.showGetStarted" class="catalogs-project-summary-list">
+    <div ng-if="$ctrl.maxDisplayProjectsReached" class="projects-count-area">
+      <span class="counter-text">{{$ctrl.maxDisplayProjects}}</span>
+      <span>of</span>
+      <span class="counter-text">{{$ctrl.totalProjects}}</span>
+      <span>Projects</span>
+      <a href="#" class="projects-view-all" ng-click="$ctrl.showAllProjects()">View All</a>
+    </div>
+    <div id="catalogs-projects-summary-list">
+      <div class="catalogs-edit-project-panel" ng-if="$ctrl.showEditProjectPanel" style="top: {{$ctrl.editPanelTopPosition}}px;">
+        <div class="arrow"></div>
+        <div class="catalogs-create-project-title">
+          <span>Edit Project - </span>
+          <span>{{$ctrl.edittingProject | displayName}}</span>
+          <a class="pull-right" ng-click="$ctrl.closeEditProjectPanel()">
+            <span class="catalogs-create-project-close pficon pficon-close"></span>
+          </a>
+        </div>
+        <edit-project project="$ctrl.edittingProject" is-dialog="true" alerts="$ctrl.alerts" redirect-action="$ctrl.onEditProject" on-cancel="$ctrl.closeEditProjectPanel"></edit-project>
+      </div>
+      <div ng-repeat="project in $ctrl.projects track by $index" class="project-tile">
+        <div class="dropdown  dropdown-kebab-pf" uib-dropdown="">
+          <button class="btn btn-link uib-dropdown-toggle" type="button" id="dropdownKebab" aria-haspopup="true" aria-expanded="true" uib-dropdown-toggle>
+            <span class="fa fa-ellipsis-v"></span>
+          </button>
+          <ul class="uib-dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebab">
+            <li><a href="#" ng-click="$ctrl.editProject(project, 'catalogs-summary-project-' + $index)">Edit</a></li>
+            <li>
+              <delete-link
+                  kind="Project"
+                  resource-name="{{project.metadata.name}}"
+                  project-name="{{project.metadata.name}}"
+                  display-name="{{(project | displayName)}}"
+                  type-name-to-confirm="true"
+                  stay-on-current-page="true"
+                  alerts="$ctrl.alerts">
+              </delete-link>
+            </li>
+          </ul>
+        </div>
+        <div id="{{'catalogs-summary-project-' + $index}}" class="project-tile-header">
+          <span ng-if="project.statusIconClass" class="project-status {{project.statusIconClass}}"></span>
+          <a href="#" ng-click="$ctrl.handleProjectClick(project)" class="project-title">{{project | displayName}}</a>
+          <span class="project-date">{{project.metadata.creationTimestamp | date: 'MMM d yyyy'}}</span>
+        </div>
+        <div class="project-description">
+          {{project | description}}
```

Comment:
Yes, they are limited to one line here. Waiting on final designs on the list items